ocs_ci/utility/rgwutils.py

```diff
@@ -14,9 +14,9 @@ def get_rgw_count(ocs_version, is_upgrade, version_before_upgrade):
     Otherwise, RGW Count is 1
 
     Args:
-        ocs_version (str): OCS Version
```

Comment:
Maybe add new type to unit test as well?

Reply:
This isn't a change in functionality, just updating the pydoc to reflect that you can pass a float or a string. We would basically be testing the `float()` function but if we feel more comfortable with them then I can add tests in that use floats instead of strings.